packages/jest-types/src/Circus.ts

```diff
@@ -208,6 +208,7 @@ export type State = {
   originalGlobalErrorHandlers?: GlobalErrorHandlers;
   parentProcess: Process | null; // process object from the outer scope
   rootDescribeBlock: DescribeBlock;
+  seenDone: boolean;
```

Comment:
We probably want this property on a single test/hook, not on the global state, sorry for not catching it earlier

Reply:
Oh, noted. Should I store the variable above this? https://github.com/facebook/jest/blob/d63f18a80909db1c1c2ee29e599ed1b03fad583d/packages/jest-circus/src/run.ts#L145-L153 ?

Reply:
Stick it on `Circus.TestEntry` and `Circus.Hook` when they are created, defaulting to `false`, then flip to `true` where you do today